crates/admin/src/rest_api/version.rs

```diff
@@ -15,7 +15,7 @@ use restate_core::TaskCenter;
 use restate_types::config::Configuration;
 
 /// Min/max supported admin api versions by the server
-pub const MIN_ADMIN_API_VERSION: AdminApiVersion = AdminApiVersion::V2;
+pub const MIN_ADMIN_API_VERSION: AdminApiVersion = AdminApiVersion::V3;
```

Comment:
Maybe add a comment why we bumped the min admin version to V3 and in which Restate version this happened.